lib/pdfkit/pdfkit.rb

```diff
@@ -63,7 +67,7 @@ def to_pdf(path=nil)
     invoke = command(path)
 
     result = IO.popen(invoke, "wb+") do |pdf|
-      pdf.puts(@source.to_s) if @source.html?
+      @sources.each{ |source| pdf.puts(source.to_s) if source.html? }
```

Comment:
Did a testing on this branch. This doesn't actually solve the multiple pages issue properly

[foo.pdf](https://github.com/pdfkit/pdfkit/files/3640698/foo.pdf)

Reply:
For supporting multiple sources, we might need to create temp files & feed it to wkhtmltopdf instead of reading it from STDIN

Reply:
Added few changes on top of this PR & got it working as expected
https://github.com/pdfkit/pdfkit/compare/master...selvagsz:feature/multiple_sources?expand=1#diff-281cf1c6253ac3b28df6c300d042a10eR38

[foo_after_fix.pdf](https://github.com/pdfkit/pdfkit/files/3640735/foo_after_fix.pdf)

Reply:
Hey @selvagsz thanks for testing my branch. I'm wondering how are you testing this, since I've been using this since 2016 in one of my projects and it has been working since always without needing to add temp files on the process.

Reply:
@JonaMX tried passing array of html strings like below
```ruby
kit = PDFKit.new(['<html><body><h1>Foo</h1></body></html>', '<html><body><h1>Bar</h1></body></html>'])
kit.to_file('./tmp/combined.pdf')
```

Reply:
I'd like to push this feature forward. Would be happy to pair/help